app/controllers/items_controller.rb

```diff
@@ -3,8 +3,12 @@ class ItemsController < ApplicationController
   def index
    if current_merchant?
      @items = current_user.items
+     @top_five_items = Item.five_popular('desc')
```

Comment:
You should be able to DRY this up by moving @top_five_items and @bottom_five_items below the if/else/end. That way it'll be available regardless of user type.

Reply:
Nice! I'll add this!